pycaret/internal/plots/utils/time_series.py

```diff
@@ -0,0 +1,357 @@
+import warnings
+from typing import Optional, List, Tuple, Any
+
+import numpy as np
+import pandas as pd
+import matplotlib
+import matplotlib.pyplot as plt
+
+
+import plotly.express as px
+import plotly.graph_objects as go
+
+from statsmodels.graphics.gofplots import qqplot
+from statsmodels.tsa.stattools import pacf, acf
+
+from sktime.transformations.series.difference import Differencer
+
+
+# TODO: Move to Time Series Utils (not plot utils)
+def _reconcile_order_and_lags(
+    order_list: Optional[List[Any]] = None, lags_list: Optional[List[Any]] = None
+) -> Tuple[List[int], List[str]]:
+    """Reconciles the differences to lags and returns the names
+    If order_list is provided, it is converted to lags_list.
+    If lags_list is provided, it is uses as is.
+    If none are provided, assumes order = [1]
+    If both are provided, returns empty lists
+
+    Parameters
+    ----------
+    order_list : Optional[List[Any]], optional
+        order of the differences, by default None
+    lags_list : Optional[List[Any]], optional
+        lags of the differences, by default None
+
+    Returns
+    -------
+    Tuple[List[int], List[str]]
+        (1) Reconciled lags_list AND
+        (2) Names corresponding to the difference lags
+    """
+
+    return_lags = []
+    return_names = []
+
+    if order_list is not None and lags_list is not None:
+        msg = "ERROR: Can not specify both 'order_list' and 'lags_list'. Please specify only one."
+        warnings.warn(msg)  # print on screen
+        return return_lags, return_names
+    elif order_list is not None:
+        for order in order_list:
+            return_lags.append([1] * order)
+            return_names.append("Order=" + str(order))
```

Comment:
Can we use f-strings instead of concatenation?

Reply:
Done